### PR TITLE
Feature/e UI 8425

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8425

### Change description ###
Form validation errors

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by DavidJayakumar — APPROVED** [review]
LGTM

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.html [hunk 1]
```diff
@@ -19,52 +21,56 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-4">{{ 'Query details' | rpxTran
...
+            <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }} </span>{{ 'File error message' | rpxTranslate }}
           </p>
 
           <input class="govuk-file-upload" id="add-documents" name="add-documents" type="file">
         </div>
       </div>
-    </div>
+    </form>
+    <button class="govuk-button" data-module="govuk-button">
+      {{ 'Next' | rpxTranslate}}
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.html:63]: @MateiHMCTS - minor point on spacing of `rpxTranslate` and `}}`; same on Line 69

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - really minor point on formatting but otherwise OK

**Review by Daniel-Lam — DISMISSED** [review]
@MateiHMCTS - looks good now; approved

**Review by DavidJayakumar — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-management-error-messages/query-management-error-messages.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div *ngIf="errorMessages.length > 0" class="govuk-error-summary"
+     aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+  <h2 class="govuk-error-summary__title" id="error-summary-title">
+    {{'There is a problem' | rpxTranslate}}
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-management-error-messages/query-management-error-messages.component.html:4]: space after and before braces {{ }}

**Review by DavidJayakumar — COMMENTED** [review]
@MateiHMCTS Please see my comments

**Review by DavidJayakumar — COMMENTED** [review]
@MateiHMCTS Please see my comments

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.ts [hunk 3]
```diff
@@ -24,4 +28,16 @@ export class WriteQueryManagementFieldComponent extends AbstractFieldReadCompone
     this.queryItem = new QueryListItem();
     Object.assign(this.queryItem, partyMessagesMockData[0].partyMessages[0]);
   }
+
```
[the remark this answers is not in the document]
Comment (MateiHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.ts:31]: True; Forgot about this as this was done a while ago.

**Pull request closed without merging**